tailwind.config.ts

```diff
@@ -42,8 +42,8 @@ module.exports = {
   ],
 
   daisyui: {
-    themes: ["light"], // false: only light + dark | true: all themes | array: specific themes like this ["light", "dark", "cupcake"]
-    darkTheme: "synthwave", // name of one of the included themes for dark mode
+    themes: ["light", "night"], // false: only light + dark | true: all themes | array: specific themes like this ["light", "dark", "cupcake"]
+    darkTheme: "night", // name of one of the included themes for dark mode
```

Comment:
```suggestion
    themes: ["light", "synthwave"], // false: only light + dark | true: all themes | array: specific themes like this ["light", "dark", "cupcake"]
    darkTheme: "synthwave", // name of one of the included themes for dark mode
```

`synthwave` is our dark theme, not `night`.